.github/workflows/build.yaml

```diff
@@ -11,18 +11,18 @@ defaults:
     shell: bash
 
 env:
-  ARM_TOOLS_URL: https://armkeil.blob.core.windows.net/developer/Files/downloads/gnu-rm/9-2020q2/gcc-arm-none-eabi-9-2020-q2-update-x86_64-linux.tar.bz2
-  ARM_TOOLS_DIR: gcc-arm-none-eabi-9-2020-q2-update
+  ARM_TOOLS_URL: https://developer.arm.com/-/media/Files/downloads/gnu-rm/10-2020q4/gcc-arm-none-eabi-10-2020-q4-major-x86_64-linux.tar.bz2
+  ARM_TOOLS_DIR: gcc-arm-none-eabi-10-2020-q4-major
 
 jobs:
   setup-build-matrix:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```

Comment:
updated to 20 since it seems like we're telling people to use 20 now? @younesr1  @cindyli-13

Reply:
Yea I think we should move towards ubuntu 20, I also think it should be synced with sw too, I'm hoping we can have more fw members also contributing to sw in the future. We're switching to ros noetic right?

Reply:
TBD but either noetic or ros2 foxy will put us on 20.04

Reply:
either. ie. we def running with ubuntu 20 as our main os